### PR TITLE
React Native 50 update

### PR DESCRIPTION
**Pull request by Vladimir-Vdovic**
Updated dependencies for forks and out-of-date packages.

Changed `Image` component to be compatible with RN 0.50.0 `ImageBackground` component.

Removed obsolete prop `resizeMode` from `theme.js`.

### CODE REVIEW

**Comment by sky-franciscogoncalves** [comment]
Hi @Vladimir-Vdovic,

I tried to add your PR to my project and pointed both themes and animations to your rn-50-update PRs. However, I was not able to get my app running as PropTypes error was presented.

 Do you have any recommendation on how to get this working?

**Comment by Vladimir-Vdovic** [comment]
Hey @sky-franciscogoncalves,

Just use `npm i shoutem/ui#feature/rn-50-update --save` and you should be good to go. Theme and Animation have been released.

File: components/DropDownMenu/DropDownModal.js [hunk 1]
```diff
@@ -212,6 +213,10 @@ class DropDownModal extends Component {
       color: bufferColor
     }];
 
+    if(!NativeModules.BVLinearGradient){
```
Comment (dodsky) [inline components/DropDownMenu/DropDownModal.js]: Put it on top of the function.

File: package.json [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
+    "@shoutem/theme": "~0.11.0",
     "babel-plugin-transform-decorators-legacy": "~1.3.4",
     "buffer": "~4.5.1",
     "events": "1.1.0",
     "html-entities": "~1.2.0",
     "htmlparser2": "~3.9.0",
     "lodash": "~4.15.0",
     "qs": "~4.0.0",
-    "react-native-lightbox": "shoutem/react-native-lightbox#4e58b9b1aca166142d69b161ff511efc3358ff9f",
+    "react-native-lightbox": "shoutem/react-native-lightbox#master",
```
Comment (dodsky) [inline package.json]: Don't point to master directly, add a tag to both repos and point to tag. This might get broken if someone commits to master.

**Comment by JulianKingman** [comment]
@Vladimir-Vdovic I'm getting resizeMode warnings, is that related to this PR? How to fix?

simulator screen shot - iphone x - 2018-01-08 at 16 07 15

**Pull request merged**